scripts/v2/create-kind-wi-storage.sh

```diff
@@ -42,24 +44,26 @@ if [ -f "$DIR/azure/rg.txt" ]; then
   exit 0
 fi
 
+ISSUER_URL="https://${AZURE_STORAGE_ACCOUNT}.z22.web.core.windows.net/"
```

Comment:
Is this `z22` deterministic? When I just tried out these steps manually mine happens to be `z13`.

Reply:
Thats a good question... it may not be fixed let me see if I can get it dynamically

Reply:
Here's a command that shows me my URL if that's helpful:
```
az storage account show --name "${AZURE_STORAGE_ACCOUNT}" -o json | jq -r .primaryEndpoints.web
```

Reply:
Fixed this, thanks!